### PR TITLE
ravioli method

### PR DESCRIPTION
**Pull request by MaximilianKing**
Password instead of ip checking for post and pop requests from pis.

### CODE REVIEW

File: server/application.py [hunk 1]
```diff
@@ -119,15 +118,17 @@ def reset_jobs():
     completed = queue.Queue(maxsize=20)  # a queue of recently completed jobs
 
 
-def check_ip(ip):
-    print(ip)
-    return ip in application.config["PI_IPS"]
+def check_password(pword):
+    if pword not in application.config["FLASK_PASS"]:
```
Comment (ishmandoo) [inline server/application.py:122]: Will there be multiple passwords?

File: server/application.py [hunk 2]
```diff
@@ -119,15 +118,17 @@ def reset_jobs():
...
+        print("Bad password from ip:{}".format(request.remote_addr))
```
Comment (ishmandoo) [inline server/application.py]: Smart

File: server/application.py (hunk 1 above)
Comment (ishmandoo) [inline server/application.py]: This looks like it's expecting multiple passwords.

**Review by ishmandoo — CHANGES_REQUESTED** [review]
Sloppy work, but I'll approve it for now.

**Review by ishmandoo — APPROVED** [review]

File: server/application.py (hunk 1 above)
Comment (MaximilianKing) [reply server/application.py]: Resolved

**Pull request merged**